internal/storage/fake/testing/bucket_tests.go

```diff
@@ -4458,3 +4456,110 @@ func (t *cancellationTest) ReadObject() {
 				HasSubstr("request canceled"))))
 	ExpectLt(time.Since(before), 50*time.Millisecond)
 }
+
+func (t *listTest) TestListForUnsupportedNames() {
+	AssertEq(
+		nil,
+		createEmpty(
+			t.ctx,
+			t.bucket,
+			[]string{
+				"//",
+				"a/b",
```

Comment:
Sorry `a/b` is not valid object?
a could be directory.

Reply:
obsolete comment. code removed.